packages/orbit-components/src/primitives/ButtonPrimitive/common/createRel.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type { CreateRel } from "./createRel";
+
+const createRel: CreateRel = ({ rel, href, external }) => {
+  const relValues = rel ? rel.split(" ") : [];
+  // add noopener and noreferrer whenever external
+  if (external && href) {
+    if (!relValues.includes("noopener")) {
+      relValues.push("noopener");
+    }
+    if (!relValues.includes("noreferrer")) {
+      relValues.push("noreferrer");
+    }
+  }
+  return relValues.length > 0 ? relValues.join(" ") : undefined;
+};
+
```

Comment:
consider to cover it with test :)

Reply:
I've just noticed that the coverage of tests of Button, ButtonLink and ButtonPrimitive is very low, so we should introduce more tests regardless this change 🙁

Reply:
then it can be done by separate MR i think :)

Reply:
I'll put it into Jira – and connect it with move to `testing-library/react`